packages/manager/apps/web/client/app/hosting/cron/hosting-cron.controller.js

```diff
@@ -16,7 +25,9 @@ export default class HostingCronsCtrl {
     HostingCron,
     WucUser,
   ) {
+    this.$rootScope = $rootScope;
```

Comment:
Nitpick: is it possible to void $rootScope? it will be easy for upgrade to another framework from AngularJs